From: Martin Pitt <martinpitt@gnome.org>
Date: Tue, 7 Jan 2014 09:40:01 +0100
Subject: keymap: Recognize different Toshiba Satellite capitalizations

https://launchpad.net/bugs/665918
(cherry picked from commit c79af123de72cdb70f122c7fda5b2401acc5ea7b)
---
 hwdb/60-keyboard.hwdb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index d253724..84a061f 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -1056,9 +1056,9 @@ keyboard:dmi:bvn*:bvr*:bd*:svnFOXCONN:pnQBOOK:*
 keyboard:dmi:bvn*:bvr*:bd*:svnMTC:pn*:pvrA0:*
 keyboard:dmi:bvn*:bvr*:bd*:svnMio*Technology:pnN890:*
 keyboard:dmi:bvn*:bvr*:bd*:svnPEGATRON*CORP.:pnSpring*Peak:*
-keyboard:dmi:bvn*:bvr*:bd*:svnTOSHIBA:pnSatellite*[uU]30[05]*:pvr*
+keyboard:dmi:bvn*:bvr*:bd*:svnTOSHIBA:pnSatellite*[uU][35]0[05]*:pvr*
+keyboard:dmi:bvn*:bvr*:bd*:svnTOSHIBA:pnSATELLITE*[uU][35]0[05]*:pvr*
 keyboard:dmi:bvn*:bvr*:bd*:svnTOSHIBA:pnSatellite*Pro*[uU]300*:pvr*
-keyboard:dmi:bvn*:bvr*:bd*:svnTOSHIBA:pnSATELLITE*[uU]500*:pvr*
 keyboard:dmi:bvn*:bvr*:bd*:svnViooo*Corporation:pnPT17:*
 keyboard:dmi:bvn*:bvr*:bd*:svnHANNspree:pnSN10E100:*
 keyboard:dmi:bvn*:bvr*:bd*:svnGIGABYTE:pni1520M:*
